### PR TITLE
Prep for upcoming ofi updates

### PR DESCRIPTION
**Pull request by dennisklein**

### CODE REVIEW

**Review by rbx — APPROVED** [review]
Solid "Alexey-PR" :)

File: test/message/_message.cxx [hunk 1]
```diff
@@ -1,223 +1,232 @@
...
-    ASSERT_EQ(reinterpret_cast<uintptr_t>(msg1->GetData()) % alignment, 0);
+    for (auto const align : {Alignment{64}, Alignment{32}}) {
+        size_t const size25{25};
+        size_t const size50{50};
 
-    size_t alignment2 = 32;
-    FairMQMessagePtr msg2(push.NewMessage(25, fair::mq::Alignment{alignment}));
-    msg2->Rebuild(50, fair::mq::Alignment{alignment2});
-    ASSERT_EQ(reinterpret_cast<uintptr_t>(msg2->GetData()) % alignment2, 0);
+        auto msg(push.NewMessage(size25));
```
Comment (rbx) [inline test/message/_message.cxx:167]: Actually this is wrong. Before the test did (1) Rebuild from default alignment to specific alignment, (2) Rebuild from specific alignment to a different specific alignment.

Comment (rbx) [reply test/message/_message.cxx:167]: I'll fix it in my own branch, as I am extending some of the tests here.

Comment (dennisklein) [reply test/message/_message.cxx:167]: Thx

**Pull request merged**